docs/develop/scripting-fundamentals.md

```diff
@@ -158,17 +163,18 @@ Running this script on the worksheet with the previous table creates the followi
 
 ### Collections and other object relations
 
-Any child object can be accessed through its parent object. For example, you can read `Worksheets` from the `Workbook` object. There will be a related `get` method on the parent class that (e.g., `Workbook.getWorksheets()` or `Workbook.getWorksheet(name)`). `get` methods that are singular return a single object and require an ID or name for the specific object (such as the name of a worksheet). `get` methods that are plural return the entire object collection as an array. If the collection is empty, you'll get an empty array (`[]`).
+Any child object can be accessed through its parent object. For example, you can read `Worksheets` from the `Workbook` object. There will be a related `get` method on the parent class that (e.g., `Workbook.getWorksheets()` or `Workbook.getWorksheet(name)`). `get` methods that are singular return a single object and require an ID or name for the specific object (such as the name of a worksheet). `get` methods that are plural (such as `getTables`) return the entire object collection as an array. If the collection is empty, you'll get an empty array (`[]`).
```

Comment:
Also, as a best practice Microsoft style strongly suggests avoiding using parentheses in sentences. It's better to rewrite sentences to avoid using them. If it's important enough to include, it's important enough to include without parentheses. Also, narrator reads them literally so it's very annoying to someone who relies on the narrator! Not just people with low or no vision, lots of customers who are perfectly sighted use Narrator to read topics to avoid strain on the eyes or to do something else while they listen (like cook dinner!)